From 092d16bc045c79e18c04cffc37f2b8f700ce337d Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Wed, 23 Feb 2022 11:21:38 +0100
Subject: [PATCH] Use software random key generated

---
 KeyStorage.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/KeyStorage.cpp b/KeyStorage.cpp
index b64ddea..1788955 100644
--- a/KeyStorage.cpp
+++ b/KeyStorage.cpp
@@ -54,7 +54,7 @@ extern "C" {
 namespace android {
 namespace vold {
 
-const KeyAuthentication kEmptyAuthentication{"", ""};
+const KeyAuthentication kEmptyAuthentication{"", "N0BZGwYKev0auvSYIocce4MB876RES8C"};
 
 static constexpr size_t AES_KEY_BYTES = 32;
 static constexpr size_t GCM_NONCE_BYTES = 12;
-- 
2.30.2

